### ISSUE TITLE
One account - many sites / Browse entry to use

### ISSUE THREAD

**Issue by piegamesde** [smorks/keepasshttp-connector#31]
At my university, you log in with your account (same username/password) on many different sites. For each site I now have to add another link to the entry, or, because multiple links are not supported (I think), clone the entry (while linking it to the original) and change the url. This is not nice.

I'd rather have an option to be able to right-click on the form, right click on it and tell it which account to use with a "browse"-function and the plugin handles the rest so that it will recognize the site the next time. Is something like this possible?

**Comment by smorks** [comment]
i agree, there really isn't a good way to do this. i'll see what i can do to make this a better experience.

**Comment by Ivan0xFF** [comment]
The "official" way of doing this is having multiple entries in your KeePass file with different URLs. 

Clone and entry and make sure to tick the "Replace username and password with references" box, then change the URL is the new entry.

**Comment by yan12125** [comment]
I have an incomplete patch for keepassxc to support multiple URLs in an entry at https://github.com/yan12125/keepassxc/tree/wip/multiple-urls. You can use regular expressions to match multiple domain names. I just don't want to have 10+ entries for my campus login.

**Comment by MBurchard** [comment]
I have just discovered this project and already love it. As I share my Keepass database between Windows and Mac OS, I switched to KeePassXC today. This is going very well.
I would also be happy if this would be solved like in KeeFox, so you could give an entry several URLs.
